dss/events/handlers/index.py

```diff
@@ -76,44 +106,45 @@ def create_index_data(s3, bucket_name, manifest, logger):
             # of the query spec, so go with that, at least for now. Do so by substituting
             # dot for underscore in the key filename portion of the index.
             # As due diligence, additional investigation should be performed.
-            index_filename = file_info["name"].replace(".", "_")
+            index_filename = file_info[BundleFileMetadata.NAME].replace(".", "_")
```

Comment:
I opened the following issue for this:
https://github.com/HumanCellAtlas/data-store/issues/280

However, this issue was not introduced in this PR, nor does this PR do anything to make the matter worse, so the independent existence of this issue should not hold up this PR.

Thanks for raising the visibility of it, and offering a potential solution!